app/_src/gateway/install/docker/index.md

```diff
@@ -251,7 +250,7 @@ docker run -d --name kong-gateway \
 
     You should receive a `200` status code.
 <!-- vale on -->
-1. (Not available in OSS) Verify that Kong Manager is running by accessing it
+1. Verify that Kong Manager is running by accessing it
```

Comment:
❗ For reviewers: The Jira ticket feedback specifically called out fixing "OSS install" instructions, I'm assuming that's when you click **Switch to OSS** at the top, but wasn't sure. And I didn't see any difference in the "Enterprise" and "OSS" versions of the docs. Did I miss anything? Did I miss an OSS install page?

Reply:
@cloudjumpercat , I also don't see a change happening w/ the Switch to OSS button but you can force the switch by appending` /?install=oss`.

All of the Linux install options have an OSS version, so please make changes there too.